common/app/views/fragments/head.scala.html

```diff
@@ -27,16 +27,9 @@
     @fragments.stylesheets(projectName, content.exists(_.tags.isCrossword), content.exists(_.tags.isInteractive))
 }
 
-@*
-Protect against ReferenceErrors in IE:
```

Comment:
This can actually be removed completely, we don't support any browsers that require it. (JS/CSS is a mess in IE9, so it's not even worth keeping just in case)

![image](https://user-images.githubusercontent.com/638051/85113636-85f68280-b20f-11ea-895b-9b7be2dfae76.png)

Reply:
Ah ok, maybe one for a follow up PR.